### ISSUE TITLE
Client tries to auto upload after account has been deleted

### ISSUE THREAD

**Issue by methyphobia** [nextcloud/android#5506]
Android client tries to auto upload files to an account that has been deleted in the client. At least it shows an upload notification that won't disappear until the app is forcefully stopped (see screenshot below). 
This behaviour was reported and fixed almost three years ago (#942), but possibly it's back ;)

### Steps to reproduce
1. Set up an account on the android client and activate auto upload for a folder (Whatsapp Images in my case)
2. Delete the whole account from your nextcloud client (without disabling auto upload prior to the deletion)
3. See a new upload notification when a file is added to the folder that had auto upload enabled

### Expected behaviour
Client shouldn't try to auto upload files to a deleted account.

### Actual behaviour
Client tries to auto upload although the account has been deleted.

### Environment data
Android version: 10

Device model: Google Pixel 3a

Stock or customized system: stock

Nextcloud app version: 3.10.1

Nextcloud server version: 18.0.1

#### Web server error log
```
none, since the web server doesn't exist anymore
```

#### Nextcloud log (data/nextcloud.log)
```
none, since the web server doesn't exist anymore
```

Clearing the app cache did _not_ have any effect on this issue.

![Screenshot_20200222-194930](https://user-images.githubusercontent.com/32457692/75097719-43e56900-55ae-11ea-906f-ccbedf522a2b.png)

**Comment by tobiasKaminsky** [comment]
Is this still happening after a force-restart of the app?

**Comment by methyphobia** [comment]
Yes, it is. I did force-close the app several times since it was the only way to get rid of the upload notification.

I solved it for me by setting up a new (dummy) instance under the former url, added the account to my android client again and disabled auto-upload. However, although the account has previously been deleted from the android client, the old auto-upload settings were still there after setting up the new account to my dummy instance.
